@meshgen/meshgen.m

```diff
@@ -782,16 +784,17 @@
                 
                 %7. Bring outside points back to the boundary
                 d = feval(obj.fd,p,obj,[],1); ix = d > 0;                  % Find points outside (d>0)
-                ix(1:nfix)=0;
-                if sum(ix) > 0
-                    dgradx = (feval(obj.fd,[p(ix,1)+deps,p(ix,2)],obj,[])...%,1)...
+                ix(1:nfix) = 0;

```

Comment:
In general, if the points aren't projecting nicely back into the domain, one simple solution is to execute the project points back twice per meshing iteration.

Reply:
It is working fine.